Source/SubtitleProvider.h

```diff
@@ -213,6 +213,42 @@ class SubtitleProvider :
         SetSubtitleDelay(newDelay);
     }
 
+    int parseInt(std::wstring const& str)
```

Comment:
I just noticed: I moved those methods from SubtitleProvider.h to SubtitleProviderSsaAss.h on winui branch, because the methods are not relevant for other types of subs. I think you accidentially added them back. Also your fix on the hex values is probably not used. You shoud apply it at SubtitleProviderSsaAss.h and remove the methods here.

Reply:
> I just noticed: I moved those methods from SubtitleProvider.h to SubtitleProviderSsaAss.h on winui branch, because the methods are not relevant for other types of subs. I think you accidentially added them back.

There were a few commits that I didn't have in my working branch. If your move was a recent change, then I might not have had these changes and that's probably why I added them back (and forgot about it later).